include/fmt/std.h

```diff
@@ -113,7 +113,7 @@ void write_escaped_path(basic_memory_buffer<Char>& quoted,
 
 }  // namespace detail
 
-FMT_EXPORT
+FMT_BEGIN_EXPORT
 template <typename Char> struct formatter<std::filesystem::path, Char> {
```

Comment:
Considering that clang and MSVC accept export on partial specializations, I don't think we should make this and similar changes until there is clarity which one is correct.

Reply:
Looks like this is the relevant paper than made export of partial specializations ill-formed: https://www.open-std.org/jtc1/sc22/wg21/docs/papers/2022/p2615r0.html.

Reply:
I think the correct fix is to remove `FMT_EXPORT` without wrapping in `FMT_BEGIN_EXPORT` / `FMT_END_EXPORT`.

Reply:
OK. I'll take your word for it as I'm not sure I'm qualified enough to be 100% certain it's not going to break anything. I didn't get anywhere trying to build the test-module.cc and only have a very minimal test project to test these changes with.